apps/journeys-admin/src/components/Editor/VideoLibrary/VideoFromLocal/LocalDetails/LocalDetails.tsx

```diff
@@ -78,12 +79,21 @@ export function LocalDetails({
     })
   }
 
+  const handleDisplay = (): void => {
+    displayMore ? setDisplayMore(false) : setDisplayMore(true)
+  }
+
   const time = data?.video?.variant?.duration ?? 0
   const duration =
     time < 3600
       ? new Date(time * 1000).toISOString().substring(14, 19)
       : new Date(time * 1000).toISOString().substring(11, 19)
 
+  const videoCaption =
```

Comment:
```suggestion
  const videoDescription =
```

Just staying true to the data inside it